src/main/java/com/mattermost/integration/figma/webhook/WebhookController.java

```diff
@@ -24,10 +23,11 @@ public WebhookController(FileNotificationService fileNotificationService) {
     }
 
     @PostMapping("/comment")
-    public void comment(@RequestBody FileCommentWebhookResponse response) {
-        if (Objects.nonNull(response) && !response.getEventType().equals("PING")) {
+    public void comment(@RequestBody FileCommentWebhookResponse response) throws Throwable {
```

Comment:
I believe you want change RequestBody Obect  type here :) cause you don't have all data that mm provides to you in request

Reply:
nvm my fault